src/mail/editor/SendMailModel.ts

```diff
@@ -753,8 +760,8 @@ export class SendMailModel {
 	 * @reject {LockedError}
 	 * @reject {UserError}
 	 * @param mailMethod
-	 * @param getConfirmation
-	 * @param waitHandler: Function to call while waiting for email to send
+	 * @param getConfirmation: A callback to get user confirmation
+	 * @param waitHandler: A callback to allow UI blocking while the mail is being sent. it seems like wrapping the send call in showProgressDialog causes the confirmation dialogs not to be shown. We should fix this, but this works for now
```

Comment:
Even if it *did* work showing dialog on top of dialog is eh. Progress should be shown only when we are doing something. There are other ways to solve this too but my point is there are non-technical reasons for this.